HousingRepairsOnlineApi.Tests/ControllersTests/RepairRequestsControllerTests.cs

```diff
@@ -49,13 +54,13 @@ public async Task TestEndpoint()
 
             saveRepairRequestUseCaseMock.Setup(x => x.Execute(It.IsAny<string>(), It.IsAny<RepairRequest>())).ReturnsAsync(repair);
 
-            var result = await systemUnderTest.SaveRepair(repairTypeArgument, repairRequest);
+            var result = await systemUnderTest.SaveRepair(repairTypeArgument, repairRequest, (bookingReference, sorCode, priority, locationId, startDateTime, endDateTime, repairDescriptionText) => bookAvailableAppointmentUseCaseMock.Object.Execute(bookingReference, sorCode, priority, locationId, repairDescriptionText));
```

Comment:
The new function parameter is repeated 3 times and given it's definition is quite long, it would be best to refactor it.